src/routes/(inner)/components/autocomplete/+page.svelte

```diff
@@ -222,9 +222,9 @@ const flavorOptions: AutocompleteOption[] = [
 		<section class="space-y-4">
 			<h2>Input Chip</h2>
 			<p>We've provided a demo of using Autocomplete alongside a Skeleton Input Chip component below.</p>
-			<pre>inputChip: {JSON.stringify(inputChip, null, 2)}</pre>
-			<pre>flavorOptions: {JSON.stringify(flavorOptions, null, 2)}</pre>
-			<pre>inputChipList: {JSON.stringify(inputChipList, null, 2)}</pre>
+			<!-- <pre>inputChip: {JSON.stringify(inputChip, null, 2)}</pre> -->
```

Comment:
Is this on accident?
If they are no longer necessary, could be removed

Reply:
I need to check the git blame on this, but someone left these in and it made it all the way to production darn it. You're right though, should just be axed. Handling that now.